eng/common/cross/toolchain.cmake

```diff
@@ -71,8 +71,17 @@ endif()
 
 if(TARGET_ARCH_NAME MATCHES "^(arm|armel)$")
   add_compile_options(-mthumb)
-  add_compile_options(-mfpu=vfpv3)
-  if(TARGET_ARCH_NAME STREQUAL "armel")
+  if (NOT DEFINED ARM_FPU_TYPE)
```

Comment:
A nit, we name other cmake variables that we pass from the command line with CLR_ prefix. I would prefer to keep this convention for these variables too.